### PR TITLE
FeatureExample:

### PR DESCRIPTION
**Pull request by dljones79**
This was as example of how to create a feature branch.

### CODE REVIEW

**Comment by zlittj** [comment]
This is so dope!

**Comment by MWilson03** [comment]
//no comment lol

**Comment by MWilson03** [comment]
i don't like that you didn't convert that to a string resource

**Comment by carlisle316** [comment]
I don't like that you didn't convert that to a string resourde

**Comment by skywalker22** [comment]
sit back and watch the forking magic

**Comment by AndrewJamesOtt** [comment]
I'm sorry dave

File: SourceTreeExample/app/src/main/res/layout/content_main.xml [hunk 1]
```diff
@@ -15,5 +15,5 @@
     <TextView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!" />
+        android:text="@string/michael_called_me_out_like_a_punk" />
```
Comment (dljones79) [inline SourceTreeExample/app/src/main/res/layout/content_main.xml:18]: Here you go buddy.  I swapped it over.

**Comment by MWilson03** [comment]
good job dave

File: SourceTreeExample/.idea/misc.xml [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+      <value>
+        <list size="4">
+          <item index="0" class="java.lang.String" itemvalue="org.jetbrains.annotations.Nullable" />
+          <item index="1" class="java.lang.String" itemvalue="javax.annotation.Nullable" />
+          <item index="2" class="java.lang.String" itemvalue="edu.umd.cs.findbugs.annotations.Nullable" />
+          <item index="3" class="java.lang.String" itemvalue="android.support.annotation.Nullable" />
+        </list>
+      </value>
+    </option>
+    <option name="myNotNulls">
```
Comment (AndrewJamesOtt) [inline SourceTreeExample/.idea/misc.xml]: #batman

**Pull request merged**